api/DocumentOrShadowRoot.json

```
@@ -390,8 +390,7 @@
               "prefix": "webkit"
             },
             "safari_ios": {
-              "version_added": true,
-              "prefix": "webkit"
+              "version_added": false
```

The [Safari 12 release notes](https://developer.apple.com/documentation/safari_release_notes/safari_12_release_notes#3126426) (and [this tweet](https://twitter.com/rmondello/status/1041739671775043596) from someone at Apple), suggests the Fullscreen API is supported but only on the iPad. I'm suggesting we go a slightly different route here

```suggestion
              "version_added": "12",
              "alternative_name": "webkitFullscreenElement",
              "partial_implementation": true,
              "notes": "Full-screen mode is only supported on the iPad."        
```

What do you think?